internal/explorer/explorer.go

```diff
@@ -21,12 +21,14 @@ type Resource struct {
 	ReadOp        *high.Operation
 	UpdateOp      *high.Operation
 	DeleteOp      *high.Operation
+	Parameters    []*high.Parameter
```

Comment:
Can we name this something more explicit like `CommonParameters`?

Same suggestion for the `DataSource` struct below